build/BranchInfo.props

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(IsStableProject)' != 'true'">
     <MajorVersion>0</MajorVersion>
-    <MinorVersion>15</MinorVersion>
+    <MinorVersion>16</MinorVersion>
     <PatchVersion>0</PatchVersion>
```

Comment:
ML.NET was released at `1.3.1` & `0.15.1` for preview. So this preview version should match at `0.15.1`.

After we release, we'll bump our version to 0.16.0. Same as ML.NET did after releasing ([PR](https://github.com/dotnet/machinelearning/pull/4069)).

Ordering:
* Set AutoML/CLI version to 0.15.1
* Update the AutoML/CLI to use ML.NET `1.3.1`
* _(release)_
* Update AutoML/CLI version to 0.16.0

Reply:
Thanks.

#resolved